pwnlib/qemu.py

```diff
@@ -90,6 +90,9 @@ def archname():
     Returns the name which QEMU uses for the currently selected
     architecture.
 
+.. doctest::
```

Comment:
Why are we skipping this test here? And only for powerpc?  It's doing a dict lookup.